GoogleUtilities/Environment/third_party/GULHeartbeatDateStorage.h

```diff
@@ -18,17 +18,31 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-/// Stores a date to a specified file.
-@interface FIRCoreDiagnosticsDateFileStorage : NSObject
+// Stores either a date or a dictionary to a specified file.
```

Comment:
nit: The comment looks outdated. Also an Apple Docs single line comment should start from `///`. I think it can be fixed in a followup PR.